src/main/java/ch/njol/skript/conditions/CondIsOnline.java

```diff
@@ -33,32 +33,41 @@
  * @author Peter Güttinger
  */
 @Name("Is Online")
-@Description("Checks whether a player is online.")
+@Description("Checks whether a player is online. The 'connected' version will return false once this player leaves the server, even if they rejoin.")
 @Examples({"player is online",
-		"player-argument is offline"})
+	"player-argument is offline",
+	"while player is connected:",
+	"\twait 60 seconds",
+	"\tsend \"hello!\" to player"
+})
 @Since("1.4")
 public class CondIsOnline extends PropertyCondition<OfflinePlayer> {
 	
 	static {
-		register(CondIsOnline.class, "(online|1¦offline)", "offlineplayers");
+		register(CondIsOnline.class, "(online|:offline|:connected)", "offlineplayers");
 	}
 	
+	private boolean connected; // https://github.com/SkriptLang/Skript/issues/6100
+	
```

Comment:
Is the comment here necessary?

Reply:
It's just an explainer to future me as to why we added a second pattern/method for it.